designer/client/__tests__/page-create.jest.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { initI18n } from "../i18n";
+import { render } from "@testing-library/react";
+import PageCreate from "../page-create";
+
+describe("page create", () => {
+  initI18n();
```

Comment:
in `jest-setup.js`, I've added a change which imports and inits i18n, so you shouldn't need to do this here

Reply:
I am cleaning up those on another branch